resources/styles/nheko-dark.qss

```diff
@@ -177,12 +176,25 @@ UserInfoWidget {
     border-bottom: 1px solid #202228;
 }
 
-#UserSettingScrollWidget > QComboBox {
+QPushButton,
+QComboBox,
+QAbstractItemView,
+QComboBox > * {
+    background: white;
     color: #202228;
 }
 
-#UserSettingScrollWidget > QComboBox {
-    color: #202228;
+QComboBox:hover,
+QAbstractItemView::item:hover,
+QPushButton:hover {
+    background: #38a3d8;
+    selection-background-color: #38a3d8;
+}
+
+
+QAbstractItemView::item:selected {
+    border-color: darkblue;
```

Comment:
these seem to make inconsistent styles where in some cases the system theme is used, and sometimes this one.

Reply:
Something that came to mind just now, I can't really do something about the wrong theme being used. I think I actually overrode the theme in most places, but the dropdown and context menus use different Views for the lists and you basically can't theme the comboboxes... I don't think I can easily fix that ._.